src/c++/vdo/base/dm-vdo-target.c

```diff
@@ -1086,18 +1106,6 @@ static void vdo_io_hints(struct dm_target *ti, struct queue_limits *limits)
 	 * The value is used by dm-thin to determine whether to pass down discards. The block layer
 	 * splits large discards on this boundary when this is set.
 	 */
-#ifndef VDO_UPSTREAM
-#undef VDO_USE_ALTERNATE
-#if defined(RHEL_RELEASE_CODE) && defined(RHEL_MINOR) && (RHEL_MINOR < 50)
-#if (RHEL_RELEASE_CODE < RHEL_RELEASE_VERSION(10, 0))
-#define VDO_USE_ALTERNATE
-#endif
-#else /* !RHEL_RELEASE_CODE */
-#if (LINUX_VERSION_CODE < KERNEL_VERSION(6, 10, 0))
-#define VDO_USE_ALTERNATE
-#endif
-#endif /* !RHEL_RELEASE_CODE */
-#endif /* !VDO_UPSTREAM */
 #ifdef VDO_USE_ALTERNATE
```

Comment:
This will need to be adjusted to align with the kernel version modification in how VDO_USE_ALTERNATE is defined. However, I do not believe that we build vdo-devel against anything with less than a 6.10 kernel, so this VDO_USE_ALTERNATE block may not be needed.

Reply:
I removed the block completely since it is not necessary anymore